websocket_test.go

```diff
@@ -120,8 +126,12 @@ func TestConcurrentClose(t *testing.T) {
 				return
 			}
 
-			go c.Write(msg)
-			go c.Close()
+			go func() {
+				_, _ = c.Write(msg)
+			}()
+			go func() {
+				_ = c.Close()
+			}()
```

Comment:
What's going on here?

Reply:
Making linters happy and making it clear that we're explicitly ignoring the errors. I can't actually _check_ the errors because this is a stress-test and both of these functions may or may not error.